roles/keycloak/defaults/main.yml

```diff
@@ -110,12 +115,20 @@ keycloak_keystores: "/srv/keycloak-keys"
 #          auth: "false"
 #          user: ""
 #          password: ""
+#        attributes:
+#          shortVerificationUri: "https://{{ rciam_hostname }}/device"
+#          # Needed attributes for metrics plugin
+#          amsUrl: "https://msg-devel.argo.grnet.gr/v1/projects/AAIMETRICS/topics/metrics"
+#          X-Api-Key: "xxx"
+#          tenenvId: "1"
+#          source: "Keycloak"
+#          keycloakUrl: "https://localhost:{{ keycloak_bind_port }}/auth"
```

Comment:
Change to "https://{{ rciam_hostname }}/{{ keycloak_base_url ==> "defaults to auth" or to "keycloak_XXXX?"